src/modules/showGrade.js

```diff
@@ -0,0 +1,14 @@
+import grades from './grades.js';
+
+const showGrade = async (tbody, url) => {
+  const records = await grades(url);
+  tbody.innerHTML = '';
+  records.result.forEach((data) => {
+    const gradeElements = ` <tr>
+    <td scope="row">${data.user}:</td>
+         <td>${data.score}</td>
+    </tr>`;
+    tbody.innerHTML += gradeElements;
+  });
```

Comment:
[OPTIONAL]
- [ ] Nice work getting the data form the API :clap: However It would be nice to make the table scroll-able for good user experience